dozer-ingestion/src/connectors/mysql/binlog.rs

```diff
@@ -655,3 +656,174 @@ impl<'a> BinlogRowsEvent<'a> {
         matches!(&self.0, DeleteRowsEventV1(_) | DeleteRowsEvent(_))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::BTreeMap;
+
+    use dozer_types::{
+        json_types::JsonValue,
+        types::{Field, FieldType},
+    };
+    use mysql_async::Column;
+    use mysql_common::{
+        binlog::{
+            jsonb::{self, JsonbString, JsonbType, OpaqueValue},
+            row::BinlogRow,
+            value::BinlogValue,
+        },
+        constants::ColumnType,
+        io::ParseBuf,
+        Value,
+    };
+
+    use crate::connectors::mysql::conversion::{IntoField, IntoFields};
+
+    #[test]
+    fn test_row_conversion() {
+        use ColumnType::*;
```

Comment:
For this test, it's very hard to see what is actually compared to what, and therefore, I can't see what is being tested. Ideally, you'd have a function
```rust
fn check(binlog_value: BinlogValue, column: Colum, expected: Field) {
// Some asserts
}
```
and the test just looks like
```rust
check(Jsonb(Null), MYSQL_TYPE_JSON, JsonValue::Null);
check(Jsonb(I16(2)), MYSQL_TYPE_JSON, JsonValue::Number(2.0.into()))
```
There, you can directly see that the test is correct.

The same goes for some of the other tests

Reply:
Agreed. Let me change it.